internal/postprocessor/main.go

```diff
@@ -280,23 +341,24 @@ func docURL(cloudDir, importPath string) (string, error) {
 	return "https://cloud.google.com/go/docs/reference/" + mod + "/latest/" + pkgPath, nil
 }
 
-func (c *config) AmendPRDescription(ctx context.Context) error {
+func (c *config) AmendPRDescription(ctx context.Context) ([]string, error) {
 	log.Println("Amending PR title and body")
 	pr, err := c.getPR(ctx)
 	if err != nil {
-		return err
+		return []string{}, err
```

Comment:
For slices, always return nil in cases like these.

Reply:
Happens more below.

Reply:
Done